apiserver/admin.go

```diff
@@ -307,7 +312,11 @@ func (a *admin) fillLoginDetails(result *authResult, lastConnection *time.Time)
 		if result.anonymousLogin {
 			logger.Debugf("anonymous model login")
 		} else {
-			logger.Debugf("model login: %s for %s", a.root.entity.Tag(), a.root.state.ModelTag().Id())
+			model, err := a.root.state.Model()
```

Comment:
We should really add the model itself to a.root just like is done for state.
This is easily done in newAPIHandler() and means that here and elsewhere model will be available without checking for an error